pkgs/applications/misc/qownnotes/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchgit, qt5 }:
+
+let
+  version = "17.02.06";
+in
+stdenv.mkDerivation {
+  name = "qownnotes-${version}";
+
+  # needs git submodules, so can't use fetchFromGitHub
+  src = fetchgit {
+    rev = "refs/tags/linux-b2804-185423";
+    url = "https://github.com/pbek/QOwnNotes";
+    sha256 = "0fknc6ajnzcdsmxd9q15i23v2lbpcp3scgq51izclkg8yd9i9lys";
+  };
+
+  buildInputs = [ qt5.qtbase qt5.qtmultimedia qt5.qtscript ];
+
+  configurePhase = ''
```

Comment:
In this project the `.pro` file is located in `src`. Does `qmakeHook` have an option to deal with that?

The project also has a cmake build setup which also needs tweaking because the translations try to install in `qt-base`.


```
-- Install configuration: "Release"
CMake Error at src/cmake_install.cmake:44 (file):
  file cannot create directory:
  /nix/store/zacdil7n5av96y85rp0l74hnxw29kk4z-qtbase-5.6.2/translations.
  Maybe need administrative privileges.
Call Stack (most recent call first):
  cmake_install.cmake:37 (include)
```